sig-rktnetes/README.md

```diff
@@ -0,0 +1,17 @@
+# The Mission
+`sig-rktnetes` is focused on the topics related to Kubernetes integration of container runtime rkt which is referred to as rktnetes. 
```

Comment:
nit: 'is focused on topics related to the integration between Kubernetes and the rkt container runtime, more succinctly referred to as "rktnetes".'